src/_zkapauthorizer/lease_maintenance.py

```diff
@@ -17,10 +17,12 @@
 refresh leases on all shares reachable from a root.
 """
 
+from __future__ import annotations
```

Comment:
This lets a classmethod on LeaseMaintenanceConfig say it returns a LeaseMaintenanceConfig.